test/grpc/test_nexus.js

```diff
@@ -685,6 +685,32 @@ describe('nexus', function () {
       });
     });
 
+    it('should change ANA state to inaccessible', (done) => {
+      client.setNvmeAnaState(
+        {
+          uuid: UUID,
+          ana_state: enums.NVME_ANA_INACCESSIBLE_STATE
+        },
+        (err, res) => {
```

Comment:
IMHO, you can just use 'done' here directly and pass result/error to it directly:
```
client.setNvmeAnaState(
        {
          uuid: UUID,
          ana_state: enums.NVME_ANA_INACCESSIBLE_STATE
        },
        done
)
```

Reply:
Thanks, fixed this and another similar existing instance nearby.